core/src/main/scala/io/finch/Output.scala

```diff
@@ -195,13 +195,13 @@ object Output {
     /**
      * Converts this [[Output]] to the HTTP response of the given `version`.
      */
-    def toResponse[CT <: String](implicit
+    def toResponse[CT](accept: Seq[Accept] = Seq.empty)(implicit
```

Comment:
Ugh. I totally forgot we need to propagate `accept` here. It's been on review for too long (and I appreciate your patience!) so let me just grab this branch, fix the problem I found with `Bootstrap` and revert changes to `ToResponse`.

Reply:
I rebased this branch and squashed everything. It's here: #905 (will merge that when it's green).